Makefile

```diff
@@ -21,7 +21,7 @@ SRC = $(PARSER_PATH)config_parser.cc lightning_main.cc lightning_server.cc serve
 all: $(TARGET)
 
 test: LDFLAGS += $(TEST_COV)
-test: integration_test
+test: clean integration_test
```

Comment:
Could we avoid `clean`as part of `test`? That way, a user can pick when a particular task should be executed.